@xen-orchestra/fs/src/abstract.js

```diff
@@ -308,11 +308,11 @@ export default class RemoteHandlerAbstract {
     return p
   }
 
-  async copy(oldPath, newPath, { checksum = false } = {}) {
+  async copy(oldPath, newPath, { checksum = false, size } = {}) {
```

Comment:
I understand the reasoning behind this but I'm i'll at ease to introduce this new param, it feels clumsy…